packages/text-link/src/TextLink.tsx

```diff
@@ -12,7 +12,7 @@ export const TextLink = forwardRef<HTMLAnchorElement, TextLinkProps>(
 	}
 );
 
-export type TextButtonProps = ButtonHTMLAttributes<HTMLButtonElement>;
+export type TextButtonProps = BaseButtonProps;
```

Comment:
Updating this to narrow down the types of of our button variant, as per other components 👍